ush/generate_FV3LAM_wflow.py

```diff
@@ -722,6 +722,10 @@ def setup_fv3_namelist(expt_config,debug):
 
     settings["gfs_physics_nml"] = gfs_physics_nml_dict
 
+    # Update levp in external_ic_nml; this should be the only variable that needs changing
```

Comment:
@MichaelLueken I already included this change in my PR ([see here](https://github.com/ufs-community/ufs-srweather-app/pull/1174/commits/8322306ec6167aaae2267bd9a7d83e40ea3eba84)), but it doesn't seem to be showing in the diffs, which is concerning. I assume a lot of merges are causing GitHub to be confused about the state of my branch. After I merge this, hopefully it will appear correctly, but we will need to double-check.

Reply:
@mkavulich In the follow up [commit](https://github.com/ufs-community/ufs-srweather-app/pull/1174/commits/6fdc2a054dabf7d9803dcf8bb49eb70232d5d8a9), these lines were removed

Reply:
Ahh I see, thanks for spotting that. Okay after the merge the changes look as expected 👍